### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Tobby8629**
This is an experimental Github project to test my knowledge on:
- branches
- pull request
- linter

activity
- created a html, css file and added a simple style to the text

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (franclobo) [inline .gitignore:1]: - [ ] Kindly add the `./test.md` file to the .gitignore file to not push it in your repo. 👀

File: README.md [hunk 2]
```diff
@@ -1,2 +1,271 @@
-# microverse
-The first microverse github experimental project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```
Comment (franclobo) [inline README.md:7]: - [ ] Please 🙏🏽 edit each section on the README file to help the common users to use your project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,271 @@
...
+## 🛠 Built With html and css <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
```
Comment (franclobo) [inline README.md]: - [ ] Kindly add the correct languages like HTML and CSS to be appropriate to your project and have a professional README file. ✌🏽

File: README.md [hunk 4]
```diff
@@ -1,2 +1,271 @@
...
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
```
Comment (franclobo) [inline README.md]: - [ ] As I said above: Kindly add the correct languages like HTML and CSS to be appropriate to your project and have a professional README file. ✌🏽

File: README.md [hunk 5]
```diff
@@ -1,2 +1,271 @@
...
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
```
Comment (franclobo) [inline README.md]: - [ ] As I said above: Kindly add the correct languages like HTML and CSS to be appropriate to your project and have a professional README file. ✌🏽 We could delete those ones we are not using in the repository.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,271 @@
...
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
```
Comment (franclobo) [inline README.md]: - [ ] Please 🙏🏽 use your own information to replace the README template for a professional file.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,271 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
```
Comment (franclobo) [inline README.md:106]: - [ ] Please 🙏🏽 add the steps to help a common user open your project, for example:

  - Clone this repo
  - Change to the feature branch: git checkout `Hello-Microverse`
  - Open the index.html file in your browser.
  - ...

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Tobby8629, 

Good job so far!
You are doing well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct Github flow ✔️
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please 🙏🏽 create the `linter.yml` file inside the **`workflows`** folder that is inside the `.github` folder to make the linters work.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Tobby8629 , 

I am Favour 👩🏽 , your reviewer for this project milestone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 
- GitHub flow used ✔️ 
- Node modules and test.md files ignored ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**